src/properties/mod.rs

```diff
@@ -805,6 +805,8 @@ define_properties! {
   "inset-inline": InsetInline(InsetInline) shorthand: true,
   "inset": Inset(Inset) shorthand: true,
 
+  "border-spacing": BorderSpacing(BorderSpacing),
```

Comment:
I think this could just be `Size2D<Length>` rather than needing a custom `BorderSpacing` type. `Size2D` already handles the parsing and omitting of the second value if it is equal to the first.

Reply:
😂😂 Done